README.md

```diff
@@ -1,2 +1,10 @@
-# helm-charts
-Helm charts for StackRox Kubernetes Security Platform
+# Helm charts for the StackRox Kubernetes Security Platform
+
+To use Helm charts to install the [StackRox Kubernetes Security Platform](https://www.stackrox.com/platform/)
```

Comment:
To use Helm charts to install Sensor, Collector and Admission controller components of the StackRox Kubernetes Security Platform ... [cc: @gaurav-nelson ]